### ISSUE TITLE
[Bug]: Links in readthedocs pages in dark mode are not coloured

### ISSUE THREAD

**Issue by migueldivo** [canonical/vanilla-framework#5460]
**Describe the bug**

In dark mode of readthedocs pages, hyperlinked text is coloured the same way as normal text. The colour becomes the typical hyperlink blue once the user has clicked on the link. This make it hard to see links that are within paragraphs and sentences.

In light mode, the hyperlinked text is coloured blue making it visible that its a link regardless if the user clicks on it. 

**To Reproduce**

1. Go to https://canonical-charmcraft--2085.com.readthedocs.build/en/2085/tutorial/write-your-first-kubernetes-charm-for-a-django-app/#add-an-initial-app
2. Set the page to dark mode
3. Go to section named "Add an initial app"
4. Scroll up to find the sentence with a link just above the header for 'Add an initial app'
5. Click on the link
6. Go backwards one page back to the tutorial page and see the link change colour

**Expected Behavior**
The links should be by default colour blue as it is in light mode without the need of a user clicking on it to change colour.

![Image](https://github.com/user-attachments/assets/c13e5133-c6b9-4d14-9581-a37c9770b020)

**Screenshots**

Before clicking:
![Image](https://github.com/user-attachments/assets/00b32329-370b-4fa8-b06f-4353ebde03f9)

![Image](https://github.com/user-attachments/assets/666d88ac-f8dd-4d38-a388-1c0142a515e2)

After clicking:
![Image](https://github.com/user-attachments/assets/fe7d26d7-8936-410f-8f2a-04160cfc4d9b)

![Image](https://github.com/user-attachments/assets/05b2ce48-5ecd-4860-8927-e372f9c5e4a8)

**Desktop (please complete the following information):**

- OS: Ubuntu 24.04
- Browser: Firefox
- Version: Latest/stable 134.0.2-1

**Comment by migueldivo** [comment]
Added to correct repo [canonical/canonical-sphinx](https://github.com/canonical/canonical-sphinx/issues/64)

**Comment by migueldivo** [comment]
Added to correct repo [canonical/canonical-sphinx](https://github.com/canonical/canonical-sphinx/issues/64)
